### PR TITLE
vmware: create initial build directory for vmware

### PR DESCRIPTION
**Pull request by kkbodke**
Initial directory to build vmware ESXi specific TDVM for ubuntu.

### CODE REVIEW

**Comment by jialeif** [comment]
Spell check error for "VMware",  "ESXi". Please append them in
https://github.com/intel/tdx-tools/blob/tdx-1.5/.github/workflows/.aspell.en.pws

File: build/ubuntu-22.04/vmware/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+1. Build all packages for ESXi TD guest
+
+```
+cd tdx-tools/build/ubuntu-22.04/vmware
+
+./build-vmware-td.sh
+```
+
+`build-vmware-td.sh` will build guest packages into  guest_repo/ .
```
Comment (jialeif) [inline build/ubuntu-22.04/vmware/README.md:15]: 2 spaces between into and guest_repo/ .
So you will build build/ubuntu-22.04/vmware/guest_repo and kernel separately,

**Review by kenplusplus — COMMENTED** [review]
Hi How about move vmware directory to <tdx-tools>/build/vmware at same level of ubuntu-22.04 and rhel, because vmware build may self-contained include host an guest on vmware distro? Do you agree? @kkbodke @jialeif

**Comment by kkbodke** [comment]
> Hi How about move vmware directory to /build/vmware at same level of ubuntu-22.04 and rhel, because vmware build may self-contained include host an guest on vmware distro? Do you agree? @kkbodke @jialeif

Hi Ken,
vmware td can be ubuntu based or rhel based.
Today it is only supporting ubuntu td.
So, I thought it would be good to place inside ubuntu directory.
The idea is that even I was thinking to move build-vmware-td.sh under build/ubuntu and have vmware directory hold just the vmware build specific configs.

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**